modules/eco-self-node-remediation-operator-about.adoc

```diff
@@ -69,13 +75,14 @@ kind: SelfNodeRemediationTemplate
 metadata:
 creationTimestamp: "2022-03-02T08:02:40Z"
 generation: 1
-name: self-node-remediation-default-template
+name: self-node-remediation-<remediation_object>-deletion-template <1>
```

Comment:
Should this YAML example have tab indentations? Please consult with an SME or QE reviewer for clarification on this. Thanks!

Reply:
Good catch, yes please (but yaml uses spaces, no tabs)
I would also skip some unimportant metadata fields, which are mostly used internally only, and aren't that meaningful for users, but I will leave that up to you.
```
apiVersion: self-node-remediation.medik8s.io/v1alpha1
kind: SelfNodeRemediationTemplate
metadata:
  creationTimestamp: "2022-03-02T08:02:40Z"
  name: self-node-remediation-<remediation_object>-deletion-template (1)
  namespace: openshift-operators
spec:
  template:
    spec:
      remediationStrategy: <remediation_strategy>  (2)
```

Reply:
Thank you @pneedle-rh and @slintes . Updated.